source/services/encryption/EncryptionHelper.ts

```diff
@@ -0,0 +1,68 @@
+import StorageService from '../StorageService';
+
+export interface UserInterface {
+  personalNumber: string;
+}
+
+export interface FormsInterface {
+  answers: { encryptedAnswers: string };
+  encryption: {
+    type: string;
+    publicKey: {
+      P: number;
+      G: number;
```

Comment:
maybe P and G could be renamed to be more clear what they are?

Reply:
Would like to keep the naming for common base pair G and P as most literature and examples uses P, G naming convention.